ipa/ipa_provider.py

```diff
@@ -421,6 +423,19 @@ def _terminate_instance(self):
         """Terminate the instance."""
         raise NotImplementedError(NOT_IMPLEMENTED)
 
+    def _collect_vm_info(self):
+        """
+        Gather basic info about VM
+        """
+        self.logger.info('Collecting basic info about VM')
+        client = self._get_ssh_client()
+        self.logger.info('systemd-analyze :')
+        self.execute_ssh_command(client, 'systemd-analyze')
+        self.logger.info('systemd-analyze blame')
+        self.execute_ssh_command(client, 'systemd-analyze blame')
+        self.logger.info('journalctl -b : ' )
```

Comment:
Thanks for updating the format. This line is failing build do to the extra space before `)`.